zapcli/cli.py

```diff
@@ -430,6 +430,54 @@ def report(zap_helper, output, output_format):
     console.info('Report saved to "{0}"'.format(output))
 
 
+@cli.command('test-alerts')
+@click.option('--config-file', '-c', default=None, type=str,
+              help='config file to use to INFO, IGNORE or FAIL warnings')
+@click.option('--config-url', '-u', default=None, type=str,
+              help='URL of config file to use to INFO, IGNORE or FAIL warnings')
+@click.option('--generate', '-g', is_flag=True, default=False,
+              help='generate default config file (all rules set to WARN)')
+@click.option('--info', '-i', is_flag=True, default=False,
+              help='default rules not in the config file to INFO')
+@click.option('--level', '-l', default='PASS', type=str,
+              help='minimum level to show: PASS, IGNORE, INFO, WARN or FAIL,' +
+              'use with -s to hide example URLs. Default: PASS')
+@click.option('--quiet', '-q', is_flag=True, default=False, type=str,
+              help='short output format - dont show PASSes or example URLs')
+@click.pass_obj
+def test_alerts(zap_helper, **options):
+    """Test zap alert and print the results, according to the configuration"""
+    if options['generate']:
+        if options['config_file']:
+            with zap_error_handler():
+                zap_helper.generate_config(options['config_file'])
+                sys.exit(0)
+        else:
+            console.error('missing required option config-file')
+            sys.exit(-1)
+
+    if not options['config_file']:
```

Comment:
You check this twice, so this could just go above the `if options['generate']:` so you don't need to also check within that block. Or, better yet, maybe we can just make the parameter required or have a documented default.